### ISSUE TITLE
Some photos in feed are displayed sideways

### ISSUE THREAD

**Issue by serdeltomas** [savvasdalkitsis/uhuruphotos-android#498]
photos taken in portrait mode are shown in landscape mode and stretched

**Comment by nagarrido** [comment]
Same happens to me

**Comment by redsunfex** [comment]
Same here, makes the app slightly useless. Other than that, it seems great.

**Comment by savvasdalkitsis** [comment]
hm do any of you have a photo you are willing to share to troubleshoot this? perhaps something in the exif of the file is not correct. i cannot replicate this :/

**Comment by marcel89l** [comment]
> hm do any of you have a photo you are willing to share to troubleshoot this? perhaps something in the exif of the file is not correct. i cannot replicate this :/

Sure.
The File "Screenshot1.png" is how it is displayed in my App in the feed, the File "Screenshot2.png" is how the picture looks inside the app when i open it, the File "IMG_20240109_163452.jpg" is the original picture.


![Screenshot1](https://github.com/savvasdalkitsis/uhuruphotos-android/assets/7256096/3f539366-758e-482d-9b61-345e2275e9b5)
![Screenshot2](https://github.com/savvasdalkitsis/uhuruphotos-android/assets/7256096/5ccd5764-594e-4f20-ba0a-8fe36da2aa9f)
![IMG_20240109_163452](https://github.com/savvasdalkitsis/uhuruphotos-android/assets/7256096/5034c27b-fa61-4bac-9ec2-f31ebbf1458d)

**Comment by redsunfex** [comment]
Yes, I have the same issue. But here's another clue. The same photos in the searched view appear correct.

![Screenshot_20240113_181303_Photos](https://github.com/savvasdalkitsis/uhuruphotos-android/assets/155825674/8ed24d15-4888-4e60-95b6-cf1540b0502b)


![Screenshot_20240113_181329_Photos](https://github.com/savvasdalkitsis/uhuruphotos-android/assets/155825674/e01b58c8-61ca-457c-b41e-8ca5fdaa743f)


Once you open the photo, it is correctly orientated.

**Comment by savvasdalkitsis** [comment]
hm. can either of you share the actual photo file here so i can add it to my dev build and debug it? Don't modify it in any way so that the exif metadata is unchanged

**Comment by redsunfex** [comment]
![0c8f814a5d1a0c23dbd355ca4c3319c31](https://github.com/savvasdalkitsis/uhuruphotos-android/assets/155825674/186be1c0-0e03-4201-8b08-254f772cd346)
![593e6e89ee80a0700e7733104fe08d271](https://github.com/savvasdalkitsis/uhuruphotos-android/assets/155825674/f69d2cf8-1f56-49a3-9e3a-4ee753ae390b)

Here are two images that appear sideways in the feed.

**Comment by savvasdalkitsis** [comment]
The fix is release in version 0.0.162 @marcel89l @nagarrido @serdeltomas @redsunfex

**Comment by redsunfex** [comment]
Thank you!

**Comment by redsunfex** [comment]
confirmed fixed.

**Comment by nagarrido** [comment]
Any chance to re-open the issue? Everything looks fine for jpg files, but heic files have this problem
